src/triggers/click.js

```diff
@@ -85,6 +85,11 @@ function vaadinRouterGlobalClickHandler(event) {
     return;
   }
 
+  // for a click event, the scroll is reset to the top position.
+  if (event && event.type === 'click') { 
+    window.scrollTo(0, 0); 
```

Comment:
Maybe we need to move that inside the `if` statement below: only scroll if the link click is handled in the router, i. e., is not outside the router’s base URL.